app/views/pages/admin.html.slim

```diff
@@ -0,0 +1,20 @@
+main.site-admins
+  section.section
+    header.has-text-centered
+      h1.is-size-1 Admin Members
+      p You can contact any of the following administrators on slack
+
+    main.container
+      .columns.is-multiline
+        - @admin_members.each do |member|
+          .column.is-12-mobile.is-6-tablet.is-4-desktop.is-equal-height
+            .card.has-text-centered
+              .card-image = image_tag member.data["profile"]["image_192"]
+
+              .card-content
+                h3.title
+                  = member.data["profile"]["real_name"]
+                - if member.data["profile"]["display_name"].present?
+                  span
+                    | @
+                    = member.data["profile"]["display_name"]
```

Comment:
This should be `member.username` and the if qualifier a few lines up should be removed.